receiver/prometheusreceiver/internal/otlp_metricfamily_test.go

```diff
@@ -436,13 +436,13 @@ func TestMetricGroupData_toSummaryUnitTest(t *testing.T) {
 				return
 			}
 
-			require.Equal(t, 1, len(mp.groups), "Expecting exactly 1 groupKey")
+			require.Len(t, mp.groups, 1)
 			// Get the lone group key.
-			groupKey := ""
+			groupKey := mp.getGroupKey(tt.labelsScrapes[0].labels.Copy())
```

Comment:
does this matter if groupKey is being overwritten in the loop below?

Reply:
Fixed.

Reply:
I deleted the iteration.